src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        FigureSupplier figureSupplier = new FigureSupplier(colorSupplier);
+        Figure[] figures = new Figure[6];
```

Comment:
Consider replacing the magic number `6` with a named constant to improve code readability and maintainability. This aligns with the checklist requirement to avoid magic numbers in your code.